packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js

```diff
@@ -0,0 +1,465 @@
+
+  it("should check that the footer's pseudo class takes up entire card to be clickable", () => {
+
+    cy.get('.bx--feature-card a.bx--link')
+		.then($els => {
+			const win = $els[0].ownerDocument.defaultView;
+			const after = win.getComputedStyle($els[0], 'after');
+			const positionValue = after.getPropertyValue('position');
+			const insetValue = after.getPropertyValue('inset');
+
+			expect(positionValue).to.eq('absolute')
+			expect(insetValue).to.eq('0px')
+		});
+
+		cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot('FeatureCard |clickable section', {
+    //   widths: [1280],
+    // });
+  });
+
+	it('should have image on the left and content on the right side of the card', () => {
+		
+    // TODO: currently React's FeatureCard stories aren't left aligned,
+    // update these values when they become as such
+
+		// image takes the left half
+		cy.get('.bx--image').then($image => {
+			expect($image[0].getBoundingClientRect().left).to.equal(344);
+			expect($image[0].getBoundingClientRect().right).to.equal(640);
+		});
+
+		// image takes the right half
+		cy.get('.bx--card__wrapper').then($content => {
+			expect($content[0].getBoundingClientRect().left).to.equal(640);
+			expect($content[0].getBoundingClientRect().right).to.equal(936);
+		});
+  });
+
+	it('should load the g10 theme', () => {
+    cy.visit(`/${_pathMedium}&theme=g10`);
+    cy.viewport(1280, 780);
+
+    cy.window().then(win => {
+      win.document.documentElement.setAttribute('storybook-carbon-theme', 'g10');
+
+      cy.screenshot();
+      // Take a snapshot for visual diffing
+      // TODO: click states currently not working in percy for web components
+      // cy.percySnapshot('FeatureCard medium | g10 theme', {
+      //   widths: [1280],
+      // });
+    });
+  });
+
+	it('should load the g90 theme', () => {
+    cy.visit(`/${_pathMedium}&theme=g90`);
+    cy.viewport(1280, 780);
+
+    cy.window().then(win => {
+      win.document.documentElement.setAttribute('storybook-carbon-theme', 'g90');
+
+      cy.screenshot();
+      // Take a snapshot for visual diffing
+      // TODO: click states currently not working in percy for web components
+      // cy.percySnapshot('FeatureCard medium | g90 theme', {
+      //   widths: [1280],
+      // });
+    });
+  });
+
+  it('should load the g100 theme', () => {
+    cy.visit(`/${_pathMedium}&theme=g100`);
+    cy.viewport(1280, 780);
+
+    cy.window().then(win => {
+      win.document.documentElement.setAttribute('storybook-carbon-theme', 'g100');
+
+      cy.screenshot();
+      // Take a snapshot for visual diffing
+      // TODO: click states currently not working in percy for web components
+      // cy.percySnapshot('FeatureCard medium | g100 theme', {
+      //   widths: [1280],
+      // });
+    });
+  });
+});
+
+describe('FeatureCard medium (mobile)', () => {
+  beforeEach(() => {
```

Comment:
Looking good, I think there's no need to copy over the same tests for mobile. Only thing I might keep is checking that the click functionality on the entire card still works when in mobile